UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -282,6 +288,15 @@ ParseReservedMemory (
           PlatformAcpiTable->Header.Revision = UNIVERSAL_PAYLOAD_ACPI_TABLE_REVISION;
           PlatformAcpiTable->Header.Length   = sizeof (UNIVERSAL_PAYLOAD_ACPI_TABLE);
         }
+      } else if (AsciiStrnCmp (TempStr, "smbios", AsciiStrLen ("smbios")) == 0) {

```

Comment:
I am not very clear on this requirement. Let me discuss with you offline and if it makes sense  can add it in the patchset

Reply:
Basing on offline discussion, since this is boot loader specific code (BuildFdtLib), please handle the same in your boot loader side and for edk2 bootloader implementation we will take care it.